transaction/examples/account/new_complex.rtm

```diff
@@ -0,0 +1,2 @@
+CREATE_ACCOUNT_ADVANCED
+    Tuple(Map<Tuple, Enum>(), Map<Tuple, Enum>(Tuple(Enum(0u8), "lock_fee"), Enum(0u8, Enum(2u8, Enum(0u8, Enum(0u8, Enum(0u8, NonFungibleGlobalId("resource_sim1q2atsr8kvzrkdpqe7h94jp9vleraasdw348gn8zg9g6n6g50t6hwlp:[cedebe542c3294b26810b8861e24b7941ae5d4967b926a3f1769]")))))), Tuple(Enum(0u8), "withdraw"), Enum(0u8, Enum(2u8, Enum(0u8, Enum(0u8, Enum(0u8, NonFungibleGlobalId("resource_sim1q2atsr8kvzrkdpqe7h94jp9vleraasdw348gn8zg9g6n6g50t6hwlp:[cedebe542c3294b26810b8861e24b7941ae5d4967b926a3f1769]")))))), Tuple(Enum(0u8), "create_proof"), Enum(0u8, Enum(2u8, Enum(0u8, Enum(0u8, Enum(0u8, NonFungibleGlobalId("resource_sim1q2atsr8kvzrkdpqe7h94jp9vleraasdw348gn8zg9g6n6g50t6hwlp:[cedebe542c3294b26810b8861e24b7941ae5d4967b926a3f1769]")))))), Tuple(Enum(0u8), "create_proof_by_ids"), Enum(0u8, Enum(2u8, Enum(0u8, Enum(0u8, Enum(0u8, NonFungibleGlobalId("resource_sim1q2atsr8kvzrkdpqe7h94jp9vleraasdw348gn8zg9g6n6g50t6hwlp:[cedebe542c3294b26810b8861e24b7941ae5d4967b926a3f1769]")))))), Tuple(Enum(0u8), "withdraw_non_fungibles"), Enum(0u8, Enum(2u8, Enum(0u8, Enum(0u8, Enum(0u8, NonFungibleGlobalId("resource_sim1q2atsr8kvzrkdpqe7h94jp9vleraasdw348gn8zg9g6n6g50t6hwlp:[cedebe542c3294b26810b8861e24b7941ae5d4967b926a3f1769]"))))))), Map<String, Enum>(), Enum(0u8, Enum(0u8)), Map<Tuple, Enum>(Tuple(Enum(0u8), "lock_fee"), Enum(0u8, Enum(2u8, Enum(0u8, Enum(0u8, Enum(0u8, NonFungibleGlobalId("resource_sim1q2atsr8kvzrkdpqe7h94jp9vleraasdw348gn8zg9g6n6g50t6hwlp:[cedebe542c3294b26810b8861e24b7941ae5d4967b926a3f1769]"))))))), Map<String, Enum>(), Enum(0u8, Enum(0u8)));
```

Comment:
Can you parametrize these resource addreses?

Actually if you rebase on top of https://github.com/radixdlt/radixdlt-scrypto/pull/985 it'd probably be easier from a merging perspective :)